modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,88 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM POWER8 or POWER9 processor-based systems
+
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance of IBM POWER8 or POWER9 processor-based systems
+
+On your IBM Power instance, set up:
+
+* 3 guest virtual machines for {product-title} control plane machines
+* 2 guest virtual machines for {product-title} compute machines
+* 1 guest virtual machine for the temporary {product-title} bootstrap machine
+
+[discrete]
+=== Disk storage for the IBM Power guest virtual machines
+
+* Storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization) or SSP(Shared Storage Pools).
+
+[discrete]
+=== Network for the PowerVM guest virtual machines
+
+* Virtualized by the Virtual I/O Server using Shared Ethernet Adapter
+* Virtualized by the Virtual I/O Server using IBM VNIC
+
+[discrete]
+=== Storage / Main Memory
+
+* 120GB / 16 GB for {product-title} control plane machines
+* 120GB / 8 GB for {product-title} compute machines
+* 120GB / 16 GB for the temporary {product-title} bootstrap machine
+
+[id="recommended-ibm-Power-system-requirements_{context}"]
+
+== Recommended IBM Power system requirements
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance of IBM POWER8 or POWER9 processor-based systems
+
+On your IBM Power instance, set up:
+
+* 3 guest virtual machines for {product-title} control plane machines
+* 2 guest virtual machines for {product-title} compute machines
+* 1 guest virtual machine for the temporary {product-title} bootstrap machine
+
+[discrete]
+=== Disk storage for the Power guest virtual machines
+
+* Storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization) or SSP(Shared Storage Pools).
+
+[discrete]
+=== Network for the PowerVM guest virtual machines
+
+* Virtualized by the Virtual I/O Server using Shared Ethernet Adapter
+* Virtualized by the Virtual I/O Server using IBM VNIC
+
+[discrete]
+=== Storage / Main Memory
+
+* 120GB / 32 GB for {product-title} control plane machines
```

Comment:
If I'm comparing the sections correctly, this storage/memory section is the only difference between the minimum and recommended sections.

If that's correct, you could consider just having one section for the minimum requirements, and then when you get to here, specify that 16GB is the minimum for control plane machines, but that 32GB is recommended. And 8GB is minimum for compute machines, but that 32GB is recommended.

Reply:
Leaving this as it is in IBM Z docs for consistency.